flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -330,8 +325,10 @@ static int32_t vtol_follower_control_simple(const float dT,
 
 	/* Calculate the difference between where we want to be and the
 	 * above position */
-	float horiz_error_norm = vtol_calculate_distances(cur_pos_ned,
-		hold_pos_ned, errors_ned, true);
+	vtol_calculate_distances(cur_pos_ned, hold_pos_ned, errors_ned, 0);
+
+	float horiz_error_mag = vtol_magnitude(errors_ned, 2);
```

Comment:
Ok. Or is this the important change. The fact that using vtol_calculate_distances is doing a 3d distance with normalization? Should it be, or should it also take a dimension parameter?